libs/shared/src/features/video-player/video-player.component.ts

```diff
@@ -74,6 +75,8 @@ export class VideoPlayerComponent implements OnDestroy {
     if (Capacitor.isNativePlatform()) {
       this.initialised = false;
     }
+    // Track video play event
+    this.analyticsService.trackVideoPlay(this.playerId);
```

Comment:
nit(non-blocking): currently the playerId is a randomly generated id, so it doesn't really hold any analytics value. It might be good to implement player IDs as suggested in https://github.com/e-picsa/picsa-apps/issues/241 (but not the full storage solution) so that we can track the id of the resource when passed to the video player and for logging in analytics

This could either be included in this pr (would only be a few lines of code), or as part of a follow-up